object-assign/object-assign.d.ts

```diff
@@ -4,6 +4,5 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare module "object-assign" {
-  function objectAssign(target: any, ...sources: any[]): any;
-  export = objectAssign;
+  export default function(target: any, ...sources: any[]): any;
```

Comment:
Well, according to this: [Typescrypt handbook/Modules](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require)
`export = objectAssign` is considered to be saying `objectAssign` is the default export.

Don't you agree?